solution.cpp

```diff
@@ -1,23 +1,144 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+* Copyright &copy; 2015 Hympert Nguyen hympert@csu.fullerton.edu
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)
+* ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
- */
+* Convert arabic numbers to roman numerals
+*/
 
-// TODO: `#include`s and function prototypes
 
+#include "stdafx.h"
```

Comment:
i've probably mentioned this to you in lab by now, but, this is a windows only thing -- probably best not to use it, unless for some reason you really need to :-)